models/access.go

```diff
@@ -4,7 +4,11 @@
 
 package models
 
-import "fmt"
+import (
```

Comment:
Please add Gitea copyright to header of file

Reply:
Done